doc/modules.md

```diff
@@ -0,0 +1,42 @@
+# Modules
+
+Due to space limitations on the cameras we created a module system that allows you to choose the extra functionality that you would like to add to your devices.This approach also makes it easier for developers to contribute to the project without needing to create pull requests back to the main repositiory.
+
+## Module compatibilty
+
+The module system has no built-in approach to ensuring modules are compatible with updates to the core system. It is each developer's responsibility to test and document the compatibility of their modules.
+
+## Module list
+
+| Name | Description |
+| --- | --- |
+| [ddns](https://github.com/martinbutt/openmiko-module-ddns) | Adds ddclient for dynamic DNS |
+| [fsck.vfat](https://github.com/martinbutt/openmiko-module-fsck-vfat) | Adds fsck.vfat for checking FAT partitions on the SD card |
+| [fsck.ext](https://github.com/martinbutt/openmiko-module-fsck-ext) | Adds e2fsck for checking ext2/3/4 partitions on the SD card |
+
+## Installing modules
+
+To install a module, add the `.tar.xz` to the modules folder of the SD card. It will be installed when the camera starts up.
```

Comment:
since this would allow the use of basically $unlimited space on the sd card, maybe the compression type should be less opinionated and leave it optional for the user

Reply:
That makes sense. As long as the existing image can decompress it, we should be able to support more formats.